ext/charlock_holmes/extconf.rb

```diff
@@ -50,7 +52,13 @@ def sys(cmd)
 have_library 'icuuc' or abort 'libicuuc missing'
 have_library 'icudata' or abort 'libicudata missing'
 
-$CXXFLAGS << ' -std=c++11'
+# icu4c might be built in C++11 mode, but it also might not have been
+icuconfig = `which icu-config`.chomp
+icuconfig = "#{icu4c}/bin/icu-config" if icuconfig.empty?
+if File.exist?(icuconfig) && `#{icuconfig} --cxxflags`.include?("c++11")
+  $CXXFLAGS << ' -std=c++11'
```

Comment:
Wonder if we should be checking if `$CXXFLAGS` is nil first before attempting to append to it? I just noticed this in https://github.com/brianmario/charlock_holmes/issues/118. For that matter, I wonder if we should check `$CFLAGS` as well...

Reply:
@brianmario  @mistydemeo , A friend of mine is getting Nil on $CXXFLAGS

Could you fix it, please?